packages/cli/commands/theme/marketplace-validate.js

```diff
@@ -100,20 +100,54 @@ exports.handler = async options => {
   }
 
   const displayResults = checks => {
+    const displayFileInfo = (file, line) => {
+      file &&
```

Comment:
Nit, can we use if statements here? This `&&` pattern is more common for react rendering, and is slightly confusing in this context.